installation-manager/src/main/resources/org/wildfly/core/instmgr/cli/command_resources.properties

```diff
@@ -13,16 +13,197 @@
+  When using this argument, this operation will remove from the server any custom patch previously uploaded and unsubscribe the server from the custom \
+  patch Channel.
+
+# Clone export
+installer.clone-export.description=\
+  Export the installation metadata.
+
+installer.clone-export.option.path.description=\
+  The configuration export destination is specified as a path to the remote machine's file system. This path is treated as an absolute path unless the \
+  'relative-to' attribute is used. If 'relative-to' is specified, the path is treated as relative to that specified directory. If the path ends with '.zip',\
+  it is treated as a file name where the configuration will be exported. If the path does not end with '.zip', a directory with that path will be assumed to exist,\
+  and a zip file containing the exported configuration will be created inside that directory.
+
+installer.clone-export.option.relative-to.description=\
+  The name of another previously named path, or one of the standard paths provided by the system. If 'relative-to' is provided, the value of the 'path' \
+  attribute is treated as relative to the path specified by this attribute. \
+  \n\
+  The standard paths provided by the system include: jboss.home - the root directory of the server distribution, user.home - user's home directory, user.dir - user's current working directory, \
+  java.home - java installation directory, jboss.server.base.dir - root directory for an individual server instance, jboss.server.data.dir - directory the server will use for persistent data \
+  file storage, jboss.server.log.dir - directory the server will use for log file storage, jboss.server.tmp.dir - directory the server will use for temporary file storage, and jboss.domain.servers.dir - \
+  directory under which a host controller will create the working area for individual server instances.
+
+# History
+installer.history.description=\
+  List previous installation states.
+
+installer.history.option.revision.description=\
+  Hash of an installation state. When using the revision argument, a detailed description about the changes made in that revision will be displayed.
+
+# Revert
+installer.revert.description=\
+  Reverts to a previous installation state.\
+  \n\
+  \n\
+  Prepare-Revert: This phase prepares the installation you want to revert to. The candidate installation will be prepared on the target installation temporal directory \
+  (jboss.domain.temp.dir on domain mode or jboss.server.temp.dir on standalone server mode). Once this phase is completed, no further preparations can be made. \
+  If you decide to discard this candidate installation, you can use the 'clean' operation at any time. \
+  \n\
+  \n\
+  Apply-Revert: Once the candidate installation is prepared, you can apply it by using the 'shutdown --perform-installation' command. This command \
+  will restart the server and apply the candidate installation.\
```

Comment:
This comment applies to both this an Apply-Update, and perhaps to the other phases as well.

Why does the user care about these names? 'Apply-Update' and 'Apply-Revert' feel like internal names for processes, but we're exposing them to the user. When I read this doc my mind naturally focuses on these names as something I need to remember and understand, but really the key point is I need to use the 'shutdown' command.

Please note that I am reading the commit before reading other ones, as I wanted to use it to help get my mind oriented to what the software is trying to do. So it's possible these terms are exposed to the user in other ways that  I just haven't gotten to yet.

Reply:
They are not exposed in the integration, but structuring the help by phases helps us to describe how the other options affects to the whole operation, e.x --dry-run (skip the prepare-updates phase), --confirm, skips the list-updates phase.

Reply:
Ok, good; if they are not otherwise exposed to users in the integration this just becomes a question of how best to communicate this info in the docs. In your conversations with doc writers please be sure they understand these names are not technical terms that have to be communicated. If they have other ideas about how to communicate these concepts, they shouldn't think these are required.  

One possible suggestion is just remove the -Update and -Revert from the names. Just 'Prepare' and 'Apply'. The Revert or Update is apparent from the context. It's having the -Update / -Revert appended that causes my mind to interpret these strings as precise technical terms I need to remember and not just concepts I need to understand.

Reply:
@bstansberry Good points on Update and Revert. @yersan your rephrasing for these LGTM.